### PR TITLE
radicale: fix permissions post-install, remove deprecated pytests

### LINKED ISSUE #38923: radicale: owner mismatch for config and runit service
### Is this a new report?

Yes

### System Info

Void 5.18.19_1 x86_64-musl GenuineIntel/KVM uptodate rFF

### Package(s) Affected

radicale-3.1.8_1

### Does a report exist for this bug with the project's home (upstream) and/or another distro?

_No response_

### Expected behaviour

Radicale continues to work after an update or install without a `chown radicale:radicale /etc/radicale/*`

### Actual behaviour

`xbps-install` reset the owner in `/etc/radicale/*` to `root:root`. Together with the supplied permission, `radicale` started via the `runit` service isn't allowed to read the config and fails.

### Steps to reproduce

1. Install or update radicale
2. Use the provided runit service
3. breaks

### PR DESCRIPTION
**Pull request by fanyx**
Picks up #39973

Hopefully fixes #38923

### CODE REVIEW

**Comment by classabbyamp** [comment]
the account name is fine as-is, it was grandfathered in https://github.com/leahneukirchen/xtools/blob/master/xlint#L343

**Comment by fanyx** [comment]
ah i see
then let me revert the user name

File: srcpkgs/radicale/INSTALL [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# *-*-shell-*-*
+#
+case ${ACTION} in
+post)
+	# fix permissions and owners
+    chown radicale:radicale /etc/radicale/config
+    chown radicale:radicale /etc/radicale/rights
+    chmod 644 /etc/radicale/config
+    chmod 640 /etc/radicale/rights
```
Comment (classabbyamp) [inline srcpkgs/radicale/INSTALL:9]: use tabs

Comment (fanyx) [reply srcpkgs/radicale/INSTALL:9]: oh right, autoexpand
my bad

**Pull request closed without merging**